### DISCUSSION TITLE
Masque Skins Resetting

### DISCUSSION THREAD

**Discussion by CBHastings** [SFX-WoW/Masque#210] [Issues] [answered]
For quite sometime time anytime I log or reload my UI my skin settings for my weakauras reset. Opening masque and unchecking enable and rechecking it immediately fixes the issue.

**Comment by StormFX** [comment]
Test it with only _Masque_ and _WeakAuras_ enabled. Change the settings and then do a proper reload via `/rl` or `/console reloadui`.

**Comment by CBHastings** [comment]
I apologise, I have realised the issue only occurs when using skins that come from the extra addon 'Masque: Clean Icons' - the base skins seem to work fine.

**Comment by StormFX** [comment]
It is possible that there's a syntax error in the skin file for that add-on causing it to not save. Try it with another third-party skin and see if it saves.

**Comment by CBHastings** [comment, marked as answer]
Tried with 'Masque: Apathy' and the settings saved. Seems to be an issue with Clean Icons specifically.
